src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -3480,6 +3480,88 @@ class MavlinkStreamActuatorControlTarget : public MavlinkStream
 	}
 };
 
+class MavlinkStreamActuatorOutputStatus : public MavlinkStream
+{
+public:
+	const char *get_name() const override
+	{
+		return MavlinkStreamActuatorOutputStatus::get_name_static();
+	}
+
+	static constexpr const char *get_name_static()
+	{
+	  return "ACTUATOR_OUTPUT_STATUS";
+	}
+
+	static constexpr uint16_t get_id_static()
+	{
+		return MAVLINK_MSG_ID_ACTUATOR_OUTPUT_STATUS;
+	}
+
+	uint16_t get_id() override
+	{
+		return get_id_static();
+	}
+
+	static MavlinkStream *new_instance(Mavlink *mavlink)
+	{
+		return new MavlinkStreamActuatorOutputStatus(mavlink);
+	}
+
+	unsigned get_size() override
+	{
+		return (_act_output_sub
+			&& _act_output_sub->advertised()) ? (MAVLINK_MSG_ID_ACTUATOR_OUTPUT_STATUS_LEN + MAVLINK_NUM_NON_PAYLOAD_BYTES) : 0;
+	}
+
+private:
+	uORB::Subscription *_act_output_sub{nullptr};
+        uint64_t _act_time;
+
+	/* do not allow top copying this class */
+	MavlinkStreamActuatorOutputStatus(MavlinkStreamActuatorOutputStatus &) = delete;
+	MavlinkStreamActuatorOutputStatus &operator = (const MavlinkStreamActuatorOutputStatus &) = delete;
+
+protected:
+	explicit MavlinkStreamActuatorOutputStatus(Mavlink *mavlink) : MavlinkStream(mavlink)
+	{
+	        _act_output_sub = new uORB::Subscription{ORB_ID(actuator_outputs)};
```

Comment:
You don't need to dynamically allocate this.

`uORB::Subscription _act_output_sub{ORB_ID(actuator_outputs)};